calkit/tests/models/test_pipeline.py

```diff
@@ -165,6 +166,22 @@ def test_matlabcommandstage():
     sd = s.to_dvc()
     print(sd)
     assert sd["cmd"] == 'matlab -batch "disp(\\"Hello, MATLAB!\\");"'
+    s = MatlabCommandStage(
+        name="d",
+        environment="_system",
+        command='disp("Hello, MATLAB!"); matlab_parent',
+    )
+    assert s.dvc_deps == ["test/matlab_child.m", "test/matlab_parent.m"]
+
+
+def test_matlabscriptstage():
```

Comment:
We can set this to skip if `matlab` isn't available:

```suggestion
@pytest.mark.skipif(shutil.which("matlab") is None)
def test_matlabscriptstage():
```